packages/database/sql/native-queries.sql

```diff
@@ -0,0 +1,39 @@
+-- user_count_and_trade_count_per_proposal
+-- WITH market_actors AS (
+--     SELECT 
+--       market_acct,
+--       actor_acct,
+--       COUNT(*) AS countOrders
+--     FROM 
+--       orders
+--     GROUP BY 
+--       market_acct, actor_acct
+-- ), distinct_users_by_proposal AS (
+--     SELECT
+--         proposal_acct,
+--         COUNT(DISTINCT actor_acct) AS uniqueUsersCount,
+--         SUM(countOrders) AS totalTrades
+--     FROM market_actors
+--     JOIN markets ON markets.market_acct = market_actors.market_acct
+--     GROUP BY proposal_acct
+-- )
+-- SELECT
```

Comment:
we should add this to the hasura metadata. Otherwise when you deploy to staging, if you haven't added your native query to the metadata here it will get overwritten in the pipeline